ui/app/AppLayouts/Chat/popups/community/TransferOwnershipPopup.qml

```diff
@@ -15,36 +16,40 @@ import shared 1.0
 StatusModal {
     id: root
 
-    property string privateKey
     property var store
+    property string privateKey
+    property string communityName
 
-    header.title: qsTr("Transfer ownership")
+    width: 640
+    height: 472
+    header.title: qsTr("Transfer ownership of ") + root.communityName + qsTr(" community")
```

Comment:
Nope, don't glue the string together, use `.arg()`

Reply:
done